src/styles/todoItems.css

```diff
@@ -0,0 +1,52 @@
+
+.todo-item {
+    padding: 1.5rem;
+    border-color: black;
+    border-width: 1px;
+    border-style: solid;
+}
+
+.todo-complete {
+    background-color: #4CAF50;
+}
+
+.todo-text {
+    padding: 1rem;
+    color: black;
+}
+
+.button {
+    background-color: #4CAF50; /* Green */
```

Comment:
sort all the properties ascendingly and keep the colors in a separate file and import it.

Reply:
Sorted properties alphabetically